awx/ui/test/e2e/fixtures.js

```diff
@@ -40,11 +49,22 @@ const getOrCreate = (endpoint, data, unique = ['name']) => {
     return store[lookup].then(created => created.data);
 };
 
+/* Creates an organization if one does not already exist, and returns it.
```

Comment:
In the `getOrCreate` you used something like returning the entity if already created otherwise it would create. I think it will be better to follow the same pattern and say something like: "Retrieve an organization, create it if it does not exist." Also it would make more sense when reading the name of the function.

If you like this, I would also recommend doing for the other get functions.